src/integrationTest/groovy/wooga/gradle/githubReleaseNotes/RepoLayoutPresets.groovy

```diff
@@ -61,15 +61,15 @@ class RepoLayoutPresets {
         Credentials credentials = new Credentials(repo.userName, repo.token)
         Grgit git = Grgit.clone(dir: localPath.path, uri: repo.httpTransportUrl, credentials: credentials) as Grgit
 
-        git.checkout(branch: "master")
+        git.checkout(branch: "main")
```

Comment:
I had similar tests in the changelist library and I use `repo.defaultBranch.name` instead of using a hardcoded value. The default branch name is configurable in github/git and could be anything at this point now.

Reply:
Sounds way better, I will change it.

Reply:
Done.